client/pages/editTemplate.jsx

```diff
@@ -0,0 +1,247 @@
+import React, { useEffect, useState } from "react";
+import {
+  addTemplate,
+  saveTemplateName,
+  saveTemplatePhases,
+  deleteTemplate,
+  getTemplateByID,
+  getTemplates
+} from "../utils/apiWrapper";
+import { Head, TemplateDraft } from "../components";
+import {
+  Button,
+  Row,
+  Container,
+  Alert,
+  Form,
+  FormGroup,
+  Input,
+  Label
+} from "reactstrap";
+import "../public/styles/editTemplate.scss";
+
+export default function EditTemplate() {
+  const [name, setName] = useState("");
+  const [mounted, setMounted] = useState(false);
+  const [templateID, setTemplateID] = useState("");
+  const [error, setError] = useState(false);
+  const [templates, setTemplates] = useState([]);
+  const [isInspiration, setInspiration] = useState(false);
+  const [isIdeation, setIdeation] = useState(false);
+  const [isImplementation, setImplementation] = useState(false);
+
+  const phases = ["Inspiration", "Ideation", "Implementation"];
+  const implementationIndex = 2;
```

Comment:
I know that inspiration and ideation indices aren't flagged by the linter because they're 0 and 1, but if we're going to do this, we should add globals for those two as well. I'll probably change it later to be one state with an object.

Reply:
Sounds good, I added globals for the inspiration and ideation indices for now.